from rest_framework.exceptions import APIException


class MainException(APIException):
    pass
